go/vumitools/utils.py

```diff
@@ -162,9 +167,13 @@ def get_user_api(self):
         return self.vumi_api.get_user_api(self.get_account_key())
 
     def get_conversation(self):
+        if self._conversation_cache is not None:
+            getter = lambda key: self._conversation_cache.get_model(
+                self.get_user_api().get_wrapped_conversation, key)
```

Comment:
Any reason why we use `get_wrapped_conversation` as the getter here, but `get_conversation` as the getter in `AppWorkerMixin.get_conversation` (and wrap the conversation manually afterwards)?